includes/admin/views/html-admin-page-home.php

```diff
@@ -10,8 +10,17 @@
 }
 ?>
 <div id="sensei-home-page" class="sensei-home-page">
-	<h1><?php esc_html_e( 'Sensei Home', 'sensei-lms' ); ?></h1>
-	<?php
-	require_once __DIR__ . '/html-admin-page-home-messages.php';
-	?>
+	<noscript>
+		<h1>Sensei Home</h1>
```

Comment:
Any reason for removing the i18n?

Reply:
Hey there @jom! Thanks for the review. I simply deleted originally that title and, when I readded it, I forgot to amke it translatable again. :upside_down_face: 

Fixed in [7db1a2a](https://github.com/Automattic/sensei/pull/6059/commits/7db1a2a15958af1a2b216322d4df3e5bb92f9ee9)